### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by xanderdunn** [JuliaLang/IJulia.jl#387]
In Julia 0.4 on a Mac OSX 10.11, when I run `Pkg.add("IJulia")`, this is the output I see:

```
INFO: Cloning cache of BinDeps from git://github.com/JuliaLang/BinDeps.jl.git
INFO: Cloning cache of Conda from git://github.com/Luthaf/Conda.jl.git
INFO: Cloning cache of Homebrew from git://github.com/JuliaLang/Homebrew.jl.git
INFO: Cloning cache of IJulia from git://github.com/JuliaLang/IJulia.jl.git
INFO: Cloning cache of JSON from git://github.com/JuliaLang/JSON.jl.git
INFO: Cloning cache of Nettle from git://github.com/staticfloat/Nettle.jl.git
INFO: Cloning cache of SHA from git://github.com/staticfloat/SHA.jl.git
INFO: Cloning cache of URIParser from git://github.com/JuliaWeb/URIParser.jl.git
INFO: Cloning cache of ZMQ from git://github.com/JuliaLang/ZMQ.jl.git
INFO: Installing BinDeps v0.3.19
INFO: Installing Conda v0.1.8
INFO: Installing Homebrew v0.1.17
INFO: Installing IJulia v1.1.8
INFO: Installing JSON v0.5.0
INFO: Installing Nettle v0.2.0
INFO: Installing SHA v0.1.2
INFO: Installing URIParser v0.1.1
INFO: Installing ZMQ v0.3.0
INFO: Building Homebrew
INFO: Cloning brew from https://github.com/Homebrew/homebrew.git
Cloning into '/Users/admin/.julia/v0.4/Homebrew/deps/usr'...
remote: Counting objects: 3822, done.
remote: Compressing objects: 100% (3663/3663), done.
remote: Total 3822 (delta 41), reused 563 (delta 24), pack-reused 0
Receiving objects: 100% (3822/3822), 3.28 MiB | 0 bytes/s, done.
Resolving deltas: 100% (41/41), done.
Checking connectivity... done.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  258k  100  258k    0     0   212k      0  0:00:01  0:00:01 --:--:--  212k
==> Tapping staticfloat/juliadeps
Tapped 38 formulae (105 files, 432K)
HEAD is now at 6e73af9 pdf-tools: move to homebrew/emacs
HEAD is now at 5d226de Fix patch locations for glib
INFO: Building Nettle
==> Installing dependencies for nettle: gmp
==> Installing nettle dependency: gmp
==> Downloading https://homebrew.bintray.com/bottles/gmp-6.0.0a.el_capitan.bottle.tar.gz
######################################################################## 100.0%
==> Pouring gmp-6.0.0a.el_capitan.bottle.tar.gz
🍺  /Users/admin/.julia/v0.4/Homebrew/deps/usr/Cellar/gmp/6.0.0a: 15 files, 3.2M
==> Installing nettle
==> Downloading https://homebrew.bintray.com/bottles/nettle-2.7.1.el_capitan.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring nettle-2.7.1.el_capitan.bottle.1.tar.gz
🍺  /Users/admin/.julia/v0.4/Homebrew/deps/usr/Cellar/nettle/2.7.1: 66 files, 1.7M
To relink: brew unlink nettle && brew link nettle
INFO: Building ZMQ
==> Installing zeromq32 from staticfloat/homebrew-juliadeps
==> Installing dependencies for staticfloat/juliadeps/zeromq32: libpgm
==> Installing staticfloat/juliadeps/zeromq32 dependency: libpgm
==> Downloading https://homebrew.bintray.com/bottles/libpgm-5.2.122.el_capitan.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring libpgm-5.2.122.el_capitan.bottle.1.tar.gz
🍺  /Users/admin/.julia/v0.4/Homebrew/deps/usr/Cellar/libpgm/5.2.122: 26 files, 872K
==> Installing staticfloat/juliadeps/zeromq32
==> Downloading https://juliabottles.s3.amazonaws.com/zeromq32-3.2.5.el_capitan.bottle.tar.gz
######################################################################## 100.0%
==> Pouring zeromq32-3.2.5.el_capitan.bottle.tar.gz
🍺  /Users/admin/.julia/v0.4/Homebrew/deps/usr/Cellar/zeromq32/3.2.5: 54 files, 2.3M
To relink: brew unlink zeromq32 && brew link zeromq32
INFO: Building IJulia
INFO: Recompiling stale cache file /Users/admin/.julia/lib/v0.4/Conda.ji for module Conda.
WARNING: Module BinDeps uuid did not match cache file
===============================[ ERROR: IJulia ]================================

LoadError: __precompile__(true) but require failed to create a precompiled cache file
while loading /Users/admin/.julia/v0.4/IJulia/deps/build.jl, in expression starting on line 2

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: IJulia had build errors.

 - packages with build errors remain installed in /Users/admin/.julia/v0.4
 - build the package(s) and all dependencies with `Pkg.build("IJulia")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated
```

**Comment by stevengj** [comment]
This is JuliaLang/julia#13458, which should be fixed in 0.4.1.  The workaround in Julia 0.4 is to restart Julia and re-run `Pkg.build("IJulia")` in a fresh session.

**Comment by xanderdunn** [comment]
@stevengj Thanks!
